src/routes/[deviceUID]/dashboard/+page.server.ts

```diff
@@ -14,6 +14,7 @@ export async function load({ params }) {
   const deviceUID = params.deviceUID;
   const allEvents: NotehubEvent[] = [];
   let readings: AirnoteReading[] = [];
+  let airIndoors = 0;
```

Comment:
all right, I have a different tact to take with this suggestion.

Make this variable a boolean with a default value of `false` instead of an integer